main/main.c

```diff
@@ -721,6 +721,36 @@ static PHP_INI_MH(OnUpdateMailLog)
 }
 /* }}} */
 
+/* {{{ PHP_INI_MH */
+static PHP_INI_MH(OnUpdateMailCrLfMode)
+{
+	if (new_value) {
+		const char *val = ZSTR_VAL(new_value);
+		if (ZSTR_LEN(new_value) > 0 &&
+			strcmp(val, "crlf") != 0 &&
+			strcmp(val, "lf") != 0 &&
+			strcmp(val, "mixed") != 0 &&
+			strcmp(val, "os") != 0) {
```

Comment:
This could've been `zend_string_equals_literal()` to use the higher-level zend_string APIs (making the code more readable).

Reply:
Ah yes that's cleaner. Addressed in #19759